src/modelAnalysis/FBA/optimizeCbModel.m

```diff
@@ -84,7 +84,7 @@
 %    FBAsolution:       solution object
 %
 %                          * f - Objective value
-%                          * x - Primal
+%                          * v - Reaction rates (Optimal primal variable, legacy FBAsoltion.x)
 %                          * y - Dual
 %                          * w - Reduced costs
 %                          * s - Slacks

```

Comment:
Sure. But was x was a good decision for the flux variable in the first place? All the papers I know use v for flux. We will maintain backward compatibility but pass out x and v the same.